src/utils/time.utils.ts

```diff
@@ -1,4 +1,4 @@
-export function dateFormat(date: Date | string): string {
+export function dateFormat(date: Date | string, locale: string | undefined): string {
```

Comment:
can we do:
`locale?: string`

or if we are not sure it will always come, we can set a default
`locale: string = 'he-il'` or better import it from `locales.hebrew`